airflow/www/static/js/dag/details/graph/Node.tsx

```diff
@@ -93,6 +93,11 @@ export const BaseNode = ({
     [, operatorTextColor] = labelStyle.split(":");
   }
 
+  const nodeBGColor =
```

Comment:
```suggestion
  const nodeBorderColor =
```

This is a border not a background.